runtime/src/main/java/org/corfudb/runtime/view/ClusterStatusReport.java

```diff
@@ -17,15 +17,15 @@
     /**
      * Connectivity to the node.
      */
-    enum NodeStatus {
+    public enum NodeStatus {
         UP,
         DOWN
     }
 
     /**
      * Health of the cluster.
      */
-    enum ClusterStatus {
+    public enum ClusterStatus {
```

Comment:
Also these enums are becoming part the exposed api, so regardless of the above decision maybe consider adding more comments to their java docs briefly describing each state.